README.md

```diff
@@ -26,6 +26,8 @@ gems:
 
 ## Configuration
 
+The configuration file is `_config.yml`, the same as Jekyll config file.
+
```

Comment:
`The configuration file is the same as Jekyll's, which is _config.yml uses the special key "assets"`  I know I was against linking to Jekyll's configuration but having `_config.yml` point to Jekyll's configuration page might be the perfect opportunity to do it unobtrusively (in that you'll have no complaints from me!)